### PR TITLE
MRG: Rename empty_room param to er_bids_path in write_raw_bids()

### PR DESCRIPTION
**Pull request by hoechenberger**
The is a preparation to make resolving #895 easier. Also I believe the new parameter name is now more explicit, which is good.

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [x] PR title starts with [MRG]
- [x] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: mne_bids/write.py [hunk 1]
```diff
@@ -1181,6 +1182,11 @@ def write_raw_bids(raw, bids_path, events_data=None, event_id=None,
            add support for Windows 10 at a later time.
 
     empty_room : mne_bids.BIDSPath | None
+
+        .. deprecated::
+           This parameter has been deprecated in favor of ``er_bids_path`` and
+           will be removed in a future version of MNE-Python.
```
Comment (hoechenberger) [inline mne_bids/write.py:1188]:
```suggestion
           will be removed in a future version of MNE-BIDS.
```

File: examples/anonymize_dataset.py [hunk 2]
```diff
@@ -76,7 +76,7 @@
 # Write experimental MEG data, fine-calibration and crosstalk files
 write_raw_bids(
     raw=raw, bids_path=bids_path, events_data=events_path, event_id=event_id,
-    empty_room=bids_path_er, verbose=False
+    er_bids_path=bids_path_er, verbose=False
```
Comment (agramfort) [inline examples/anonymize_dataset.py:79]: er_bids_path=bids_path_er

hum... one of the 2 needs to be changed so it's consistent

**Review by agramfort — COMMENTED** [review]
but honestly I am not convinced...

er_ is very cryptic in a parameter name and we break backward compat
unless people passed named parameters which was never enforced

so I have strong doubts on this one...

**Review by sappelhoff — COMMENTED** [review]
I don't like the `er_bids_path` too much either, I agree with @agramfort that `er` is too cryptic.

`empty_room_bids_path` is too long, but I'd find it better than `er_bids_path`.

Is this all in preparation to add a `empty_room` boolean flag in a future release for #895 ? If you are not sure yet how you are going to solve that (or somebody else), I am not sure that we should prepare something right before a release. IMHO it can wait and then we can iterate after 0.9 and before 0.10

**Comment by hoechenberger** [comment]
Fair enough, let's reconsider this sometime later.

**Pull request closed without merging**